src/packages/cli/src/Init.ts

```diff
@@ -82,6 +82,14 @@ export const defaultURL = (
   }
 }
 
+export const defaultGitIgnore = () => {
+  return `
+node_modules
```

Comment:
Does this create the file with an empty line at the beginning? If so, that is probably undesired.